src/styles/Book.css

```diff
@@ -3,3 +3,137 @@ td.book-id,
 td.remove-book {
   text-align: center;
 }
+
+.book-card {
+  width: 100%;
+  height: 170px;
+  margin-top: 30px;
+  display: flex;
+  align-items: center;
+  padding: 30px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.book-details {
+  width: 40%;
+}
+
+.book-details p {
+  cursor: pointer;
+}
+
+.category-heading {
+  margin: 8px 0;
+  opacity: 0.5;
+  font-size: 14px;
+  font-weight: bold;
+  color: #121212;
+  font-family: Montserrat, sans-serif;
+}
+
+.book-title {
+  width: 100%;
+  font-size: 22px;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+  font-family: RobotoSlab, sans-serif;
+}
+
+.book-author {
+  width: 107px;
+  height: 19px;
+  margin: 4px 0 0;
+  font-size: 14px;
+  font-weight: 300;
+  color: #4386bf;
+  font-family: RobotoSlab, sans-serif;
+}
+
+.comment-remove-edit {
+  width: 60%;
+  height: 19px;
+  font-size: 14px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-family: RobotoSlab, sans-serif;
+}
```

Comment:
These two CSS rules have repeating properties e.g. `font-weight`, `color`, `font-family`, `font-size`, and `height` are all the same. You could have them under the same selector e.g. `.book-author, .comment-remove-edit {...}`. This would make updating these styles easier since you would target only one selector. ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

Reply:
@ezeilo-su, Great job working on the previous review comment but seems we are still having the same `height` been repeated.

Perhaps the following code could help.
```
.book-author, .comment-remove-edit {
    height: 19px;
}
```